contrib/zpages/build.gradle

```diff
@@ -0,0 +1,14 @@
+description = 'OpenCensus Z-Pages'
+
+apply plugin: 'java'
+
+[compileJava, compileTestJava].each() {
+    it.sourceCompatibility = 1.8
+    it.targetCompatibility = 1.8
+}
```

Comment:
I'm not too deep into Gradle - just wondering if this is effectively the same as
sourceCompatibility = 1.8
targetCompatibility = 1.8